sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -17,21 +17,30 @@ class AggrDist(BasePairwiseTransformerPanel):
 
     Components
     ----------
-    transformer: pairwise transformer of BaseTrafoPw scitype (tabular pairwise)
+    transformer: pairwise transformer of BasePairwiseTransformer scitype
 
     Hyper-parameters
     ----------------
-
     aggfunc: aggregation function 2D np.array -> float
         default = np.mean
+    aggfunc_symm: bool - whether aggregation function is symmetric
```

Comment:
* What does it mean for aggfunc to be symmetric?
* Maybe this was meant to be `transformer_symm`, because it is the transformer that creates the matrix, not the aggfunc. (I think)
* Also, as general good practice, name of boolean variable should make it clear it is a boolean variable. something like `is_aggfunc_symm` or `aggfunc_is_symm`

Reply:
`aggfunc_symm` encodes knowledge about `aggfunc` which is in general not queriable from `aggfunc`, so I think it's sensible to pass it here?

I've changed the name of it to `aggfunc_is_symm`.

Reply:
@fkiraly

> `aggfunc_symm` encodes knowledge about `aggfunc` which is in general not queriable from `aggfunc`, so I think it's sensible to pass it here?

Sure, but that was not my question. My question is what does it even mean for an aggfunc to be symmetric?

Based on later discussions, I think it means that aggfunc(M) = aggfunc(transpose(M)). Whatever it is, I think it should be clearly stated in docstring because it is non-obvious. (e.g. symmetric might mean that the function is invariant under any permutation of entries of the matrix while preserving the shape of the matrix)

Reply:
>Sure, but that was not my question. My question is what does it even mean for an aggfunc to be symmetric?

Gives the same result if you transpose the matrix, indeed.

Reply:
I added a comment